Disable documentation build/install, it needs asciidoc
and we don't use it anyway.

Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>

diff -Nura btrfs-progs-v3.14.2.orig/Makefile btrfs-progs-v3.14.2/Makefile
--- btrfs-progs-v3.14.2.orig/Makefile	2014-05-29 21:56:59.000000000 -0300
+++ btrfs-progs-v3.14.2/Makefile	2014-06-17 16:53:29.404520306 -0300
@@ -57,8 +57,8 @@
 btrfs_fragment_libs = -lgd -lpng -ljpeg -lfreetype
 
 SUBDIRS =
-BUILDDIRS = $(patsubst %,build-%,$(SUBDIRS)) build-Documentation
-INSTALLDIRS = $(patsubst %,install-%,$(SUBDIRS)) install-Documentation
+BUILDDIRS = $(patsubst %,build-%,$(SUBDIRS))
+INSTALLDIRS = $(patsubst %,install-%,$(SUBDIRS))
 CLEANDIRS = $(patsubst %,clean-%,$(SUBDIRS))
 
 .PHONY: $(SUBDIRS)
